WooCommerce/Classes/ViewRelated/Hub Menu/HubMenuViewModel.swift

```diff
@@ -116,6 +116,8 @@ final class HubMenuViewModel: ObservableObject {
             navigationPath: navigationPathBinding)
     }()
 
+    var cardPresentPaymentService: CardPresentPaymentFacade?
```

Comment:
nit: could be private(set)

Updated in https://github.com/woocommerce/woocommerce-ios/pull/12970/commits/e4f821057c54a13f7439c02fa814574b6683e867.

```suggestion
    private(set) var cardPresentPaymentService: CardPresentPaymentFacade?
```